connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -301,6 +301,15 @@ public void timestampToConnectOptional() {
         assertNull(schemaAndValue.value());
     }
 
+    @Test
+    public void timestampToConnectDefval() {
```

Comment:
For completeness, should we have default value tests for the other logical types? (Also super nitpicky, but can you spell out "DefaultValue"?)